spec/models/vm/retirement_management_spec.rb

```diff
@@ -57,7 +60,7 @@
         vm_with_owner.retirement_check
         vm_with_owner.reload
         expect(vm_with_owner.retirement_last_warn).not_to be_nil
-        expect(vm_with_owner.retirement_requester).to eq('admin')
+        expect(vm_with_owner.retirement_requester).to eq(user.userid)
```

Comment:
The test is `"uses admin as requester"` but this change seems like we are not using the admin.